### ISSUE TITLE
Eclipse Plugin Update Site Not Found

### ISSUE THREAD

**Issue by Adrriver** [palantir/eclipse-typescript#361]
Make the get request, get 404 error.  Seems like a problem.

**Comment by jganesh** [comment]
yes same with me..!! is there any alternative way to enable typescript in eclipse?

**Comment by PyvesB** [comment]
> yes same with me..!! is there any alternative way to enable typescript in eclipse?

You can use Wild Web Developer to get proper Typescript support in Eclipse:
https://github.com/eclipse/wildwebdeveloper

@derekcicerone @sixinli maybe worth mentioning that in this project's Readme?

**Comment by Rouche** [comment]
Simple, fork the project, build and add an external site.

If the project is not updated, you might have to do it with latest dev plugin.

I myself switched to IntelliJ for full stack development.
